packages/web-components/fast-components/src/text-area/README.md

```diff
@@ -0,0 +1,4 @@
+# fast-text-area
+An implementation of an [HTML textarea element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea).
```

Comment:
Should this say "as a form-connected web-component" like the other descriptions?

Reply:
It technically has not been form connected so I haven't denoted it here. Once updated that can be added in.